osu.Game/Graphics/UserInterface/HoverSounds.cs

```diff
@@ -20,6 +22,12 @@ public class HoverSounds : CompositeDrawable
     {
         private SampleChannel sampleHover;
 
+        /// <summary>
+        /// Length of debounce for sound playback.
+        /// Set this to <see cref="TimeSpan.Zero"/> to disable debouncing.
+        /// </summary>
+        public TimeSpan DebounceTime { get; set; } = TimeSpan.FromMilliseconds(50);
```

Comment:
The game codebase currently doesn't use `TimeSpan` for everything except interop with core library. Although I think it's better to use `TimeSpan` for clearness, this PR might better use numbers for consistency.